README.md

```
@@ -1,52 +1,140 @@
+<td>
+
+**📖 프로젝트**
 - [프로젝트 소개](#-프로젝트-소개)
 - [팀원 소개](#-팀원-소개)
+- [주요 기능](#-주요-기능)
+
+</td>
+<td>
+
+**🏗️ 기술**
 - [기술 스택](#-기술-스택)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)